examples/research_projects/quantization-qdqbert/quant_trainer.py

```diff
@@ -41,8 +41,10 @@ def add_arguments(parser):
     group.add_argument("--quant-disable", action="store_true", help="disable all quantizers")
     group.add_argument("--quant-disable-embeddings", action="store_true", help="disable all embeddings quantizers")
     group.add_argument("--quant-disable-keyword", type=str, nargs="+", help="disable quantizers by keyword")
-    group.add_argument("--quant-disable-layer-module", type=str, help="disable quantizers by keyword under layer.\d+.")
-    group.add_argument("--quant-enable-layer-module", type=str, help="enable quantizers by keyword under layer.\d+.")
+    group.add_argument(
+        "--quant-disable-layer-module", type=str, help=r"disable quantizers by keyword under layer.\d+."
+    )
+    group.add_argument("--quant-enable-layer-module", type=str, help=r"enable quantizers by keyword under layer.\d+.")
```

Comment:
Those changes don't make any sense. This is docstring, there shouldn't be no `\d+` at the end and no r.